src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -185,7 +210,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
         return new FixerConfigurationResolver([
-            (new FixerOptionBuilder('elements', 'Dictionary of `const|method|property` => `none|one` values.'))
+            (new FixerOptionBuilder('elements', 'Dictionary of `const|method|property` => `none|one|one_if_phpdoc` values.'))
```

Comment:
Can we have this string generated from constants? Both `const|method|property` and `none|one|one_if_phpdoc`.

Reply:
Done